keyboards/gergo/keymaps/default/keymap.c

```diff
@@ -0,0 +1,197 @@
+/* Good on you for modifying your layout! if you don't have 
+ * time to read the QMK docs, a list of keycodes can be found at
+ *
+ * https://github.com/qmk/qmk_firmware/blob/master/docs/keycodes.md
+ *
+ * There's also a template for adding new layers at the bottom of this file!
+ */
+
+#include QMK_KEYBOARD_H
+
+#define IGNORE_MOD_TAP_INTERRUPT
+#define BASE 0 // default layer
+#define SYMB 1 // symbols
+#define NUMB 2 // numbers/motion
+
+// Blank template at the bottom
+/*  ---------- LEFT HAND -----------   ---------- RIGHT HAND ---------- */
```

Comment:
Lines 17-43 should be deleted from this file, and added to `keyboards/gergo/gergo.h`.